packages/squiggle-lang/src/rescript/Distributions/PointSetDist/Mixed.res

```diff
@@ -36,6 +36,47 @@ let updateIntegralCache = (integralCache, t: t): t => {
   integralCache: integralCache,
 }
 
+let combinePointwise = (
```

Comment:
I don't remember writing this function. 

But it's now occurring to me that maybe what's desired is for calling `klDivergence` on mixed distributions for it to split into discrete and continuous _parts_, carry out `klDivergence`, then recombine??? unclear to me.

Reply:
yea, that's what I would have expected. It's a bit messy in cases where someone didn't assign discrete probability mass to a section that really has it in the answer (leading to negative infinity), but we could later have hot fixes here